manifests/install.pp

```diff
@@ -192,7 +192,7 @@
   # Render the Shibboleth configuration. These are run time and not used
   # during the build process. It should restart Jetty, though.
   ['ldap.properties', 'idp.properties', 'authn/general-authn.xml', 'logback.xml',
-    'c14n/subject-c14n.xml', 'saml-nameid.xml',
+    'c14n/subject-c14n.xml', 'saml-nameid.xml', 'access-control.xml',
```

Comment:
So I would almost prefer to just remove that whole block which does nothing but set local variables. For some reason, I had in my head that the variables could be accessed in the template unless they were defined in the class. Obviously that's not true.

When I do remove that block, though, the linter throws warnings like this:

WARNING: top-scope variable being used without an explicit namespace on line 55

I could use the correct scope in the class, but how should the template be handled? I think that's why I started assigning local variables for things that get used in templates.